embedded/sql/stmt.go

```diff
@@ -2654,16 +2630,19 @@ func (stmt *SelectStmt) genScanSpecs(tx *SQLTx, params map[string]interface{}) (
 			if idx.sortableUsing(col.id, rangesByColID) {
 				if preferredIndex == nil || idx.id == preferredIndex.id {
 					sortingIndex = idx
+					sortingRequired = false
 					break
 				}
 			}
 		}
-
-		descOrder = stmt.orderBy[0].descOrder
 	}
 
 	if sortingIndex == nil {
-		return nil, ErrNoAvailableIndex
```

Comment:
this could potentially affects current users code, should be communicated somehow

Reply:
Updated doc: https://github.com/codenotary/immudb.io/pull/510